packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -740,6 +740,11 @@ export class GCFBootstrapper {
       const promises: Array<Promise<void>> = new Array<Promise<void>>();
       const batchSize = 30;
       for await (const installation of installationGenerator) {
+        if (installation.suspended_at !== null) {
+          // Assume the installation is suspended.
+          logger.info(`skipping installations for ${installation.id} because it is suspended`);
```

Comment:
Probably need to do this for the `schedule.installation` fanout handler too

Reply:
Sure, I moved it to eachInstallation